setup.py

```diff
@@ -1,5 +1,5 @@
 from setuptools import setup, find_packages
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
If 2.0.0 is broken does it make more sense to publish a new/non broken version of 2.0.0?